<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Abastecimiento</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family: Victor Mono
}

QFrame#frame_aux{
background: #80a7ca;
border-radius: 10px;
font-size: 13px
}

QPushButton{
border-radius: 5px;
background: #99b3ca;
font-size: 16px
}

QPushButton:hover{
background: #acc8e2;
font-weight:bold;
border-radius: 5px;
}

QComboBox{
background: #e1e1e1;
border-radius: 5px;
font-size: 15px;
}

QLabel{
font-size: 15px
}

QFrame#tabla{
background: #e1e1e1;

}

QFrame#tabla2{
background: #e1e1e1;
border-radius: 12px;
}

QFrame#frame_seleccion{
background: #99b3ca;
border-radius: 12px;
}

QLabel#titulo{
	background: #99b3ca;
	border-radius: 10px;
	font-size: 16px;
}

QFrame#frame{
background: #e1e1e1;
border-radius: 12px;
}

QLabel#label_titulo{
	background: #99b3ca;
	border-radius: 10px;
	font-size: 24px;
}

</string>
  </property>
  <widget class="QPushButton" name="btnmas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Victor Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Guardar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>560</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnmenos">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>500</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Victor Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnmas2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>500</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Victor Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnmenos2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>500</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Victor Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btniniciar">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>560</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Iniciar simulación</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabla2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>471</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabla">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>441</width>
     <height>301</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-40</y>
     <width>1361</width>
     <height>791</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/bg (Grande).jpg</normaloff>../images/bg (Grande).jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>1500</width>
     <height>1500</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="frame_aux">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>190</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad de semanas</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="iteraciones">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btncargar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar datos simulación </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btndatos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar datos simulación actual</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_seleccion">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>109</y>
     <width>651</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="tablas_opcion">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Suministro de piñas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Suminstros de consumo de vidrio</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Suministro de consumo de plastico</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Suministro de adquisicion de tapones y corchos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sumiinistros de etiquetas y sellos</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Transporte o suministro:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>851</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_titulo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>831</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>SIMULACION SECCIÓN ABASTECIMIENTO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>toolButton</zorder>
  <zorder>btnmas</zorder>
  <zorder>Guardar</zorder>
  <zorder>btnmenos</zorder>
  <zorder>btnmas2</zorder>
  <zorder>btnmenos2</zorder>
  <zorder>btniniciar</zorder>
  <zorder>tabla2</zorder>
  <zorder>tabla</zorder>
  <zorder>frame_aux</zorder>
  <zorder>frame_seleccion</zorder>
  <zorder>frame</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
